app/assets/stylesheets/bootstrap_and_overrides.scss

```diff
@@ -194,3 +194,17 @@ span.btn a{ text-decoration: none; color: #333333}
 body.modal-open {
   overflow: visible;
 }
+
```

Comment:
can you add it as a comment in code and open a issue to remove it once we upgrade please?